R/convert_sim_var.R

```diff
@@ -8,6 +8,7 @@
 #' @param \dots an expression to convert variable based on other variables or offsets
 #' @param unit the units for the new variable
 #' @param longname the longname for the new variable
+#' @param overwrite logical overwrite existing variable data?
```

Comment:
You can drop the question mark. Just say "overwriting existing variable after conversion" or something like that.